python/mxnet/symbol/contrib.py

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
why isn't there a random.py?

Reply:
It's here: https://github.com/apache/incubator-mxnet/pull/7604/files#diff-ccae178f3d50d30fce82fc1946f2d35cR1